### PR TITLE
`WorkspaceSymbolProvider` improvements

### PR DESCRIPTION
**Pull request by isc-bsaviano**
- Now includes classes as well as class members
- Only shows results from classes actually in the isfs folder (respects query params, doesn’t show anything for csp folders)
- Shows results even if no query is specified
- Uses local URI if present for client-side editing
- Resolves location correctly for delimited identifiers
- Checks cancellation token

### CODE REVIEW

File: src/providers/WorkspaceSymbolProvider.ts [hunk 1]
```diff
@@ -1,73 +1,55 @@
...
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'query' AS Type FROM %Dictionary.QueryDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'trigger' AS Type FROM %Dictionary.TriggerDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'storage' AS Type FROM %Dictionary.StorageDefinition" +
-    " UNION ALL %PARALLEL " +
-    "SELECT Name, Parent->ID AS Parent, 'projection' AS Type FROM %Dictionary.ProjectionDefinition" +
-    ") WHERE %SQLUPPER Name %MATCHES ?";
+  private readonly _sqlPreifx: string =
```
Comment (isc-rsingh) [inline src/providers/WorkspaceSymbolProvider.ts:8]: This is just cosmetic, but @isc-bsaviano did you want this variable to be called `_sqlPrefix` (not Preifx)?

Comment (isc-bsaviano) [reply src/providers/WorkspaceSymbolProvider.ts:8]: Yeah, this is intentional. I chose `_sqlPrefix` to mean "the start of the query that is always used".

Comment (isc-rsingh) [reply src/providers/WorkspaceSymbolProvider.ts:8]: I was just saying you wrote `ifx` in the code instead of `fix`

Comment (isc-bsaviano) [reply src/providers/WorkspaceSymbolProvider.ts]: Good catch, I totally missed that!

**Review by isc-rsingh — APPROVED** [review]

**Pull request merged**